.github/workflows/build.yml

```diff
@@ -0,0 +1,57 @@
+name: build
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+  workflow_dispatch:
+  
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      # required by aws-actions/configure-aws-credentials
+      id-token: write
+      contents: read
+    steps:
+    - name: run number with offset
+      env:
+        NUM: ${{ github.run_number }}
+      run: echo "GITHUB_RUN_NUMBER_WITH_OFFSET=$((10000+$NUM))" >> $GITHUB_ENV
+    - uses: actions/checkout@v3
+    - name: Setup SBT environment
+      uses: Jtalk/setup-sbt@v2
+      with:
+        version: 1.5.5
+    - name: Setup Node.js environment
+      uses: actions/setup-node@v2.5.2
+      with:
+        node-version-file: .nvmrc
+    - name: CI
+      working-directory: cdk
+      run: |
+        npm i -g yarn
```

Comment:
[Yarn is pre-installed](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md), so we shouldn't need this line.